index.html

```diff
@@ -201,7 +201,7 @@ <h1>I'm Zermarden <br>
           </div>
           <div class="card-works-4">
             <div class="snapshot-portfolio">
-              <img class="snp-p-img" src="assets/snapshot-card-4.svg" width="295px" height="220px">
+              <img class="snp-p-img" src="assets/snapshot-card-4.svg" width="295" height="220" alt="snapshot-card">
```

Comment:
- [ ] Your project images are not displaying because the image file you're referencing is not in the assets folder.